src/script/DeployLineaStateBridge.s.sol

```diff
@@ -16,15 +16,26 @@ contract DeployLineaStateBridge is Script {
     ///                            CONFIG                           ///
     ///////////////////////////////////////////////////////////////////
     string public root = vm.projectRoot();
-    string public path = "src/script/.deploy-config.json";
+    string public path = "./src/script/config.json";
     string public json = vm.readFile(path);
 
+    error ConfigError();
+
     function setUp() public {
         // https://docs.worldcoin.org/reference/address-book
 
-        messageServiceAddress = abi.decode(vm.parseJson(json, ".messageServiceAddressL1"), (address));
-        lineaWorldIDAddress = abi.decode(vm.parseJson(json, ".lineaWorldIDAddress"), (address));
-        worldIDIdentityManagerAddress = abi.decode(vm.parseJson(json, ".worldIDIdentityManagerAddress"), (address));
+        bytes memory messageServiceJson = vm.parseJson(json, ".messageServiceAddressL1");
+        bytes memory lineaWorldIDJson = vm.parseJson(json, ".lineaWorldIDAddress");
+        bytes memory worldIDIdentityManagerJson = vm.parseJson(json, ".worldIDIdentityManagerAddress");
+
+        if (messageServiceJson.length != 32 || lineaWorldIDJson.length != 32 || worldIDIdentityManagerJson.length != 32)
```

Comment:
This is done because the EVM operates on 256-bit (32-byte) words.

vm.parseJson stores the gotten address in a 32-byte memory slot padding the 20-byte address with 12 leading zero bytes to fill up the 32 bytes.